test/integration/narrow/routes/endemics/vet-rcvs.test.js

```diff
@@ -152,22 +158,27 @@ describe('Vet rcvs test when Optional PI Hunt is OFF', () => {
 describe('Vet rcvs test when Optional PI Hunt is ON', () => {
   const auth = { credentials: {}, strategy: 'cookie' }
   const url = '/claim/endemics/vet-rcvs'
+  let server
 
-  beforeAll(() => {
+  beforeAll(async () => {
     getEndemicsClaimMock.mockImplementation(() => { return { typeOfLivestock: 'pigs' } })
     setEndemicsClaimMock.mockImplementation(() => { })
     setEndemicsAndOptionalPIHunt({ endemicsEnabled: true, optionalPIHuntEnabled: true })
+    server = await createServer()
+    await server.initialize()
   })
 
-  afterAll(() => {
+  afterAll(async () => {
+    jest.resetAllMocks()
```

Comment:
Not all that important, but doing this twice in a row now

Reply:
Good catch